container/dangerzone.py

```diff
@@ -133,64 +135,72 @@ def calculate_timeout(
         return timeout
 
     async def document_to_pixels(self) -> None:
-
         conversions: Dict[str, Dict[str, Optional[str]]] = {
             # .pdf
             "application/pdf": {"type": None},
             # .docx
             "application/vnd.openxmlformats-officedocument.wordprocessingml.document": {
```

Comment:
Now that we have only a 1-1 mapping, maybe we should drop the nested dict.

Reply:
Personally, I wanted to just have three sets: LibreOffice MIME types, `convert` MIME types, and no conversion MIME types.

However, I was thinking that changing the structure completely would prohibit including more info about a MIME type in the future, that's why I left it as is. If you hard-pressed on removing the nested dict though, I can do it.

Reply:
OK. Let's keep it as-is.